ServerCore/Pages/Events/Create.cshtml.cs

```diff
@@ -70,6 +72,12 @@ public async Task<IActionResult> OnPostAsync()
 
             await _context.SaveChangesAsync();
 
+            // Create base files for event content and style

```

Comment:
Rather than have this code twice, can there just be one UploadEventFiles function in NewFileHelper?

Reply:
Not sure which twice you're referring to.  If you mean having it duplicated between Create and CreateDemo, that's by design since according to our discussion, we wanted it to be always created for real events but only optionally for demo events.  If you mean having to call this function multiple times (once for each file), I can add a simple wrapper function to do all 4 at once, but that feels unnecessary.

Reply:
I am referring to the fact that there's a lot of code duplicated for all four files, and if a fifth file appears, it'll be easy to forget one of the places. Having a single function that does all the work for both scenarios would be easier to maintain.

Reply:
That makes sense from a future-proofing standpoint.  I can add a "new event files" wrapper function.